priv/templates/jsonapi/controller.ex

```diff
@@ -0,0 +1,51 @@
+defmodule <%= module %>Controller do
+  use <%= base %>.Web, :controller
+
+  alias <%= module %>
+
+  plug :scrub_params, <%= inspect singular %> when action in [:create, :update]
+
+  def index(conn, _params) do
+    <%= plural %> = Repo.all(<%= alias %>)
+    render conn, "index.json", %{data: <%= plural %>, params: _params}
+  end
+
+  def create(conn, %{<%= inspect singular %> => <%= singular %>_params}) do
+    changeset = <%= alias %>.changeset(%<%= alias %>{}, <%= singular %>_params)
+
+    if changeset.valid? do
+      <%= singular %> = Repo.insert!(changeset)
+      render conn, "show.json", %{data: <%= singular %>}
```

Comment:
this response doesn't match JSONAPI spec http://jsonapi.org/format/#crud-creating